apps/site/assets/ts/projects/components/MoreProjectsTable.tsx

```diff
@@ -41,48 +41,53 @@ const MoreProjectsTable = ({
   });
 
   return (
-    <div>
-      <table className="c-more-projects-table" aria-label="More Projects">
-        <thead className="c-more-projects-table__thead hidden-md-down">
-          <tr>
-            <th
-              scope="col"
-              className="c-more-projects-table__th c-more-projects-table__th-project"
-            >
-              Project
-            </th>
-            <th
-              scope="col"
-              className="c-more-projects-table__th c-more-projects-table__th-last-updated"
-            >
-              Last Updated
-            </th>
-            <th
-              scope="col"
-              className="c-more-projects-table__th c-more-projects-table__th-status"
-            />
-          </tr>
-        </thead>
-        <tbody className="c-more-projects-table__tbody">
-          {state.projects.map(project => (
-            <MoreProjectsRow
-              key={project.id}
-              placeholderImageUrl={placeholderImageUrl}
-              {...project}
-            />
-          ))}
-        </tbody>
-      </table>
+    <div className="container">
+      <h2 className="hidden-sm-down featured-project-subheader">
```

Comment:
This in the elixir template was set as "className" instead of "class" so it didn't take effect. Do we want this hidden on sm down @phildarnowsky?

Reply:
@meagonqz not sure why I put that there, this header should be visible at all breakpoints.